### DISCUSSION TITLE
Using OpenStories

### DISCUSSION THREAD

**Discussion by muan** [dddddddddzzzz/OpenStories#9] [Show and tell]
Hi all! I'd like to start a thread where we just share the OpenStories implementations (and clients) that we have, so that we get 1. more feeds to follow, 2. we see how everyone's using it.

I'll start.

Where to see it: https://muan.co/
Built with [`<open-stories>`](https://github.com/dddddddddzzzz/open-stories-element) and a [custom stylesheet](https://github.com/muan/site/blob/84975b48b45cbbbfef02972bc5f1d8a90ecaa694/assets/new.scss#L597-L675).

**Comment by iam-robin** [comment]
Where to see it: https://www.iamrob.in/
Built with [`<open-stories>`](https://github.com/dddddddddzzzz/open-stories-element) and an [astro component with custom styles](https://github.com/iam-robin/personal-website/blob/main/src/components/open-stories/OpenStoriesCategory.astro)

Thanks for this awesome open source tool! 🙌

**Comment by artlung** [comment]
I added a "reels" section of my website and implemented an OpenStories jsonfeed at [artlung.com/reels/](https://artlung.com/reels/).

I expose a `<link>` with a rel alternate for it as well.
